storage/innobase/handler/ha_innodb.cc

```diff
@@ -215,6 +215,11 @@ static char*	innodb_version_str = (char*) INNODB_VERSION_STR;
 
 static my_bool	innobase_blocking_lru_restore		= FALSE;
 
+
+#ifdef UNIV_DEBUG
+static bool     srv_track_changed_pages_was_active = FALSE;
```

Comment:
Needs a comment. If InnoDB coding style allows, can be moved as static var into innodb_track_changed_pages_validate.